it/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendISpec.scala

```diff
@@ -35,24 +35,24 @@ class TraderServicesFrontendISpec
       }
     }
 
-    "GET /trader-services/route-one" should {
-      "show the enter consignment details page" in {
+    "GET /trader-services/pre-clearance" should {
```

Comment:
Should be GET  /pre-clearance-check-route-1/declaration-details ?

**Again might be worth us dropping the route-1 part, will chat with Irene if you agree